cosmic_ray/mutating.py

```diff
@@ -49,26 +49,30 @@ def activation_record(self):
         """
         return self._activation_record
 
-    def visit_mutation_site(self, node, op):
+    def visit_mutation_site(self, node, op, count):
         """Potentially mutate `node`, returning the mutated version.
 
         `Operator` calls this when AST iteration reaches a
         potential mutation site. If that site is scheduled for
         mutation, the subclass instance will be asked to perform the
         mutation.
         """
-        if self._count == self._target:
-            self._activation_record = {
-                'operator': _full_module_name(op),
-                'occurrence': self._target,
-                'line_number': cosmic_ray.util.get_line_number(node)
-            }
-
-            old_node = node
-            node = op.mutate(old_node)
-            ast.copy_location(node, old_node)
-
-        self._count += 1
+        for idx in range(count):
```

Comment:
This looks like it works fine, but the logic feels a bit heavy to me. What about something like:
```
span = self._target - self._count
if span in range(count):
    self._activation_record = ...
self._count += 1
return node
```